sim/core/spell.go

```diff
@@ -468,6 +468,14 @@ func (spell *Spell) CanCast(sim *Simulation, target *Unit) bool {
 		return false
 	}
 
+	// While moving only instant casts are possible
+	if spell.Flags&SpellFlagCanCastWhileMoving == 0 && spell.DefaultCast.CastTime > 0 && spell.Unit.Moving {
```

Comment:
This will return the cast time without the multiplier and without any modify casts being applied. The actual cast will use them so its not that big of a deal but we should fix this in a future pr

Reply:
Hm.. might be something we want to add into SOD branch as well then as I just copied this part over and added the extra check for the SpellFlag. So SoD is not using the CastMultiplier either right now